.changelog/lint-unreleased.sh

```
@@ -0,0 +1,50 @@
+#!/bin/bash
+# This script will check that the dirs in unreleased are all valid.
+
+[[ -n "$VERBOSE" ]] && set -x
+
+where_i_am="$( cd "$( dirname "${BASH_SOURCE:-$0}" )"; pwd -P )"
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)